### PR TITLE
#16 RouteSelection Rework: SVGs and custom FlatList & ScrollView components

### PR DESCRIPTION
**Pull request by mathvaillant**
### After a long time trying to make it into a single component I decided to do this "Scroll" into 2 different components (the only ones that do scroll on RN), FlatList and Scrollview. Indeed they look too similar to be split, but if we try to make a single component to work for both of them, we will end up with a messy, ugly and error prone component. Although they have some similar things, they also have pretty different behaviours and non matching props. That's the reason.

### CODE REVIEW

File: src/types/i18n/resources.d.ts
Comment (mathvaillant) [inline src/types/i18n/resources.d.ts:1]: This happened automatically..

**Comment by mathvaillant** [comment]
[Route Selection Screen Rework](https://trello.com/c/AypWKpJh/16-route-selection-screen-rework)

**Comment by joaoqueiroz95** [comment]
Just added the theme PR

**Review by joaoqueiroz95 — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]
LGTM

**Pull request merged**